File: reports/releases/v0.7.1/attempts/overhead-joern-java-attempt-01/outputs/reports/raw/invocation-overhead/joern-java/fixture/DfbTrivial.java
package dataflowbench.taint;

final class DfbTrivial {
    static int dfb_source() { // DFB-SOURCE: trivial-overhead-input
        return 1;
    }

    static void dfb_sink(int value) { } // DFB-SINK: trivial-overhead-sink

    static void run() {
        dfb_source();
        dfb_sink(0);
    }
}

File: reports/releases/v0.7.1/attempts/overhead-opentaint-kotlin-attempt-01/preexisting/reports/raw/invocation-overhead/opentaint-kotlin/fixture/DfbTrivial.kt
package dataflowbench

object DfbTrivial {
    fun dfb_source(): Int { // DFB-SOURCE: trivial-overhead-input
        return 1
    }

    fun dfb_sink(value: Int) {} // DFB-SINK: trivial-overhead-sink

    fun run() {
        dfb_source()
        dfb_sink(0)
    }
}
